Formula/k3d.rb

```diff
@@ -0,0 +1,19 @@
+class K3d < Formula
+  desc "Little helper to run Rancher Lab's k3s in Docker"
+  homepage "https://github.com/rancher/k3d"
+  url "https://github.com/rancher/k3d/archive/v1.2.1.tar.gz"
+  sha256 "eb7510aab675183b7febbad79ee23e04de6e6e47369c898ccf03ec2022045709"
+
+  depends_on "go" => :build
+
+  def install
+    system "make", "install-tools"
```

Comment:
Looking at the makefile, this is running `go get` with unpinned dependencies (on `gox` and `golangci-lint`). This is not suitable for Homebrew.

Reply:
Understood. I've removed that section from the brew recipe. Looking at it a little bit closer, that stuff is just used for CI/cross-platform builds - which we don't need here.